okapi-core/src/test/java/org/folio/okapi/ModuleTest.java

```diff
@@ -625,10 +625,22 @@ public void testFilters(TestContext context) {
 
     // Clean up (in reverse order)
     logger.debug("testFilters starting to clean up");
-    given().delete(locPostEnable).then().log().ifValidationFails().statusCode(204);
-    given().delete(locationPostDeployment).then().log().ifValidationFails().statusCode(204);
+    c = api.createRestAssured3();
+    c.given().delete(locPostEnable).then().log().ifValidationFails().statusCode(204);
+    Assert.assertTrue("raml: " + c.getLastReport().toString(),
+        c.getLastReport().isEmpty());
```

Comment:
Can you extract this duplicated code into some method, for example `assertEmptyReport(RestAssuredClient c)`?

Reply:
ok (in use in other tests, though, so could also move it to a test util).